server/store/kvstore/upgrade.go

```diff
@@ -0,0 +1,57 @@
+package kvstore
+
+import (
+	"fmt"
+
+	"github.com/blang/semver"
+)
+
+func (s *Store) UpdateDatabase() error {
+	v, err := s.System().GetVersion()
+	if err != nil {
+		return err
+	}
+	// If no version is set, set to to the newest version
+	if v == "" {
+		// TODO: Dont hardcode this
```

Comment:
`version.go`? Are you saying about `manifest.go`?
If so, I think that `manifest.go` should be in `plugin` package because of its meaning...

Would you like to add the argument `pluginVersion string` to `UpdateDatabase` function? That argument has the meaning of what version the store migrates to.
The value of `plugingVersion` is given in `OnActivation` func and via `NewStore` func.

How do you think about this?

If you don't like this idea, I'm fine that we move `manifest.go` to other package. However, in that case we don't need to change `poll` package name.

Reply:
:+1: on adding `pluginVersion string` as an argument to `UpdateDatabase`.